### PR TITLE
Edits to Managing state with field policies

### PR DESCRIPTION
**Pull request by StephenBarlow**

### CODE REVIEW

**Comment by StephenBarlow** [comment]
@hwillson This one's ready for a look next week! I made some major structural changes to the field policies article, which might be easier to read in the preview: https://deploy-preview-6516--apollo-client-docs.netlify.app/docs/react/v3.0-beta/local-state/managing-state-with-field-policies/

**Comment by StephenBarlow** [comment]
FYI I'm continuing to add edits to this open PR that extend beyond its initial scope. Everything that I add is in a state that I'm comfortable with to merge into the main PR

File: docs/source/local-state/reactive-variables.mdx [hunk 1]
```diff
@@ -42,4 +50,8 @@ cartItems([456]);
 console.log(cartItems());
 ```
 
-While reactive variables can be passed around your application and used as simple state containers, as their name implies they can also be used to trigger reactive changes in your application. Each time you call a reactive variable function with a new value, the new value is stored so it can be retrieved again in the future. At the same time the fact that a new value has been received is also broadcast to other parts of Apollo Client. Anything that has registered to receive updates when a reactive variable changes will get notified of the new data, and have a chance to act on it. We'll look at how this functionality can be combined with field policies to automatically re-run queries and dynamically update parts of your application, in [Using reactive variables to track `@client` state](./managing-state-with-field-policies/#using-reactive-variables-to-track-client-state).
+## Reacting
```
Comment (hwillson) [inline docs/source/local-state/reactive-variables.mdx:53]: Haha - I love this title.

File: docs/source/caching/cache-field-behavior.md [hunk 2]
```diff
@@ -1,18 +1,24 @@
 ---
 title: Customizing the behavior of cached fields
+sidebar_title: Customizing field behavior
 ---
 
-You can customize how individual fields in the Apollo Client cache are read and written. To do so, you define a `FieldPolicy` object for a given field. You nest a `FieldPolicy` object within whatever [`TypePolicy` object](./cache-configuration/#the-typepolicy-type)  corresponds to the type that contains the field.
+You can customize how a particular field in your Apollo Client cache is read and written. To do so, you define a **field policy** for the field. A type policy can include:
```
Comment (hwillson) [inline docs/source/caching/cache-field-behavior.md:6]: We say "field policy" then "type policy" here, which might confuse people. Maybe just make them both "field policy".

File: docs/source/data/pagination.mdx [hunk 3]
```diff
@@ -10,7 +10,7 @@ In this article, we'll cover the technical details of using Apollo to implement
 
 ## Using `fetchMore`
 
-In Apollo, the easiest way to do pagination is with a function called [`fetchMore`](../caching/cache-interaction/#incremental-loading-fetchmore), which is included in the result object returned by the `useQuery` Hook. This basically allows you to do a new GraphQL query and merge the result into the original result.
+In Apollo, the easiest way to do pagination is with a function called [`fetchMore`](../caching/advanced-topics/#incremental-loading-fetchmore), which is included in the result object returned by the `useQuery` Hook. This basically allows you to do a new GraphQL query and merge the result into the original result.
```
Comment (hwillson) [inline docs/source/data/pagination.mdx:13]: Just a quick note - we should add docs here that mention `fetchMore`'s `updateQuery` function is deprecated (https://github.com/apollographql/apollo-client/pull/6464). I'll get this done.

File: docs/source/caching/cache-field-behavior.md (hunk 2 above)
Comment (StephenBarlow) [reply docs/source/caching/cache-field-behavior.md:6]: Oops! Definitely a vestige of an iteration on my part

**Pull request merged**